packages/e2e-selenium/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+	"extends": "../../tsconfig.base.json",
+	"compilerOptions": {
+		// commented out because src folder doesn't exist in this package.
+		//"rootDir": "src",
```

Comment:
There was some prior discussion at https://github.com/WordPress/gutenberg/pull/18942#issuecomment-601772189 about folder structures for packages which aren't transpiled, using `lib` as a substitute for `src`. I believe this package in its current state is not prone to the same problems which were aimed to be addressed in having a `lib` folder (files in the root of the project), but it would be good to have the consistency all the same.

Reply:
Fixed.

Reply:
But this breaks [compressed size action](https://github.com/WordPress/gutenberg/pull/21528/checks?check_run_id=584478893).

Reply:
I decided to use `rootDirs`.